examples/test/remote/index.mjs

```diff
@@ -0,0 +1,101 @@
+    const abi = [
+    "constructor(uint256 msg)",
+  ];
+
+    //Reach Contract
+    const parentCtc = accAlice.deploy(backend);
+    console.log(`\nReach contract deploy.\n`);
+   const contractsReach = new ethers.Contract(
+   '0x42605c6B6d33932E30a1e22fc7cB123b2B41c436', 
+     abi,
+     provider.getSigner('0x42605c6B6d33932E30a1e22fc7cB123b2B41c436')
+   );
+    const logs = await provider.getLogs({
+      fromBlock: 0 ,
+      toBlock: 100,
+      address: stdlib.networkAccount,
+      topic: 'event'
+    })
+    console.log(logs);
+
+
+  
+    console.log(`\nBoth Multi-sig complete\n`);
+      
+
```

Comment:
This is almost certainly wrong because you aren't actually using the Reach program (you're not running `backend.Alice`)

Reply:
I just discovered reach’s getInfo() and backend._Connectors.ETH.ABI which is compatible w ethers,utils